scripts/make-translations.sh

```diff
@@ -0,0 +1,39 @@
+function abort () {
+    echo $@
```

Comment:
This is fine since echo "space-pads" multiple args anyway. The only caveat is if you want to preserve _exact_ whitespace (multiple spaces, tabs, newlines, etc), but that doesn't seem important here.